proto/coprocessor.proto

```diff
@@ -6,6 +6,7 @@ import "kvrpcpb.proto";
 import "gogoproto/gogo.proto";
 import "rustproto.proto";
 import "metapb.proto";
+import "span.proto";
```

Comment:
The span might be shared with TiDB as well. Maybe tipb is a better place?

Reply:
I suggest to make minitrace a standalone project and keep the proto inside the project.

Reply:
The events are business dependent (as you can see from the enum variants) and might not be a good place to put it in minitrace.

Reply:
You can define `Event event = 5` as `uint64 event = 5` and keep Event enum definition inside kvproto. kvproto is supposed to be used by TiKV, should not depend on tipb.

Reply:
Good idea!